server/models/document.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const mongoosePaginate = require('mongoose-paginate');
+
+const Schema = mongoose.Schema;
+
+const documentSchema = new Schema({
+  Title: String,
+  Content: String,
+  Owner: String,
+  Permissions: String,
+  CreatedAt: { type: Date, default: Date.now },
```

Comment:
Objects should be defined in arbnb format.
`{
type: Date,
default: Date.now(),
}`